WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -304,14 +304,14 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
 /**
  Returns device location data.
  It requires to configure location access permission by manual.
- The response is always zero (0) without authorization.
- 'authorizationStatus' indidates current authorization status.
+ The response of 'latitude' and 'longitude' are always zero (0) without authorization.
```

Comment:
and altitude

Reply:
🆗 
Tested the altitude was zero when the privacy permission was off or denied